### ISSUE TITLE
es2017.string lib value not accepted.

### ISSUE THREAD

**Issue by ryanelian** [Microsoft/TypeScript#26827]
I *vaguely* remember this option being available in earlier versions.

![image](https://user-images.githubusercontent.com/8892187/44935601-848a6a80-ad9b-11e8-9ef5-2bb9475f13d0.png)

And it has to be valid since **it actually works.**

----

On Visual Studio / Visual Studio Code

![image](https://user-images.githubusercontent.com/8892187/44935938-e9929000-ad9c-11e8-8d1b-defccd8c5eae.png)

Using `tsc`

```
src/es2017-string.ts:37:3 - error TS2339: Property 'padEnd' does not exist on type 'string'.

37 s.padEnd(1, ' ');
     ~~~~~~

src/es2017-string.ts:38:3 - error TS2339: Property 'padStart' does not exist on type 'string'.

38 s.padStart(1, ' ');
     ~~~~~~~~
```

**TypeScript Version:**  

- 3.1.0-dev.20180831
- 3.0.3
- 3.0.1

**Search Terms:**

**Code**

```ts
let s = '';
s.padEnd(1, ' ');
s.padStart(1, ' ');
```

**Expected behavior:**

tsconfig should not warn me about invalid lib?

**Actual behavior:**

tsconfig warns me about invalid lib, but did not errored!

**Playground Link:** 

**Related Issues:** 

https://github.com/Microsoft/TypeScript/issues/14793

https://github.com/Microsoft/TypeScript/issues/13396

**Comment by RyanCavanaugh** [comment]
Seems to have been caused by https://github.com/SchemaStore/schemastore/pull/520  , which in turn was caused by an inaccurate listing in the TypeScript Handbook. I'll fix both.

**Comment by RyanCavanaugh** [comment]
Opened https://github.com/SchemaStore/schemastore/pull/540 and committed an edit to the Handbook repo https://github.com/Microsoft/TypeScript-Handbook/commit/03c73b39487d03a34ad8dde53a1cdb0b18977d8b
